### PR TITLE
Support tokens for private repositories in standard mode

### PR DESCRIPTION
**Pull request by kgevorkyan**

### CODE REVIEW

File: save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt [hunk 1]
```diff
@@ -714,3 +719,19 @@ fun readStandardTestSuitesFile(name: String) =
...
+        credentialsFile.file.readText()
+            .lines()
+            .filter { it.isNotBlank() }
+            .single()
+    } else {
+        return null to null
+    }
+
+    val splitRow = fileData.split("\\s".toRegex())
+    require(splitRow.size == 2)
```
Comment (petertrr) [inline save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt:735]: Error message?

File: save-preprocessor/src/main/resources/application.properties [hunk 2]
```diff
@@ -2,6 +2,7 @@ server.port=5200
 management.endpoints.web.exposure.include=health,info,prometheus
 save.repository=/home/cnb/repositories/
 save.reposFileName=TestSuitesRepos
+save.reposTokenFileName=TestSuitesToken
```
Comment (petertrr) [inline save-preprocessor/src/main/resources/application.properties:5]: Please add an example of how to mount file from host in `docker-compose.override.yaml`

Comment (kgevorkyan) [reply save-preprocessor/src/main/resources/application.properties:5]: Did you mean
```bash
    volumes:
      - '/home/saveu/configs/TestSuitesToken:/workspace/BOOT-INF/classes/TestSuitesToken'
```
?

But we neither added such info for `TestSuitesRepo`, should we do this for `TestSuitesToken` now?

Comment (petertrr) [reply save-preprocessor/src/main/resources/application.properties:5]: Yes, please add this for both

File: save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt [hunk 3]
```diff
@@ -714,3 +719,19 @@ fun readStandardTestSuitesFile(name: String) =
                 require(splitRow.size == 2)
                 splitRow.first() to splitRow[1].split(";")
             }
+
+private fun readGitCredentialsForStandardMode(name: String): Pair<String?, String?> {
+    val credentialsFile = ClassPathResource(name)
+    val fileData = if (credentialsFile.exists()) {
+        credentialsFile.file.readText()
+            .lines()
```
[the remark this answers is not in the document]
Comment (kgevorkyan) [reply save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt]: Fixed, thanks

File: save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt (hunk 1 above)
Comment (kgevorkyan) [reply save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt]: Make sense, added

File: save-preprocessor/src/main/resources/application.properties (hunk 2 above)
Comment (kgevorkyan) [reply save-preprocessor/src/main/resources/application.properties:5]: Ok, added

**Review by petertrr — APPROVED** [review]

**Pull request merged**